wpilibc/src/main/native/include/frc/AprilTagFieldLayout.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include <wpi/SymbolExports.h>
+
+#include "frc/DriverStation.h"
+#include "frc/apriltag/AprilTagUtil.h"
+#include "frc/geometry/Pose3d.h"
+
+namespace wpi {
+class json;
+}  // namespace wpi
+
+namespace frc {
+class WPILIB_DLLEXPORT AprilTagFieldLayout {
+ public:
+  AprilTagFieldLayout() = default;
+
+  explicit AprilTagFieldLayout(
+      const std::vector<AprilTagUtil::AprilTag>& apriltags);
```

Comment:
You could add a constructor here that takes a JSON filename and calls the AprilTagUtil function for the user. In fact, that function is so short you could probably just copy-paste its contents and remove the AprilTagUtil function.